submit.kesch-test.slurm

```diff
@@ -23,8 +23,6 @@ export MALLOC_TRIM_THRESHOLD_=536870912
 export G2G=1
 export MV2_USE_GPUDIRECT=0 #with this perf are ok
 
-export LD_PRELOAD=/opt/mvapich2/gdr/no-mcast/2.2/cuda8.0/mpirun/gnu4.8.5/lib64/libmpi.so
```

Comment:
this was forgotten and is clearly wrong, so you should remove it.
@ofuhrer note that nonetheless I think I have seen some ld preload in the reframe test, so we should ask CSCS whether somehting is needed.